doc/scheduler-scripting.tex

```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
 
 \end{enumerate}
 
-\begin{figure}[htbp]
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty1}}
+		\includegraphics[scale=0.55]{images/putty1}
 	\caption{SSH tunnel configuration 1}
```

Comment:
I think `scale` is the problem for HTML render. Base it t on the `\columnwidth` or `\textwidth` instead.

Reply:
fixed